runhouse/main.py

```diff
@@ -540,7 +210,13 @@ def status(
         help="Whether to update Den with the status",
     ),
 ):
-    """Load the status of the Runhouse daemon running on a cluster."""
+    """Load the status of the Runhouse daemon running on a cluster.
```

Comment:
wondering if we should be more generic here by just saying "load the status of the cluster". not sure how important the distinction would be to a user that this is specifically checking the status of the daemon, since we make the distinction anyways between a running cluster and one where the daemon is down. wdyt?